pylab/core/models.py

```diff
@@ -56,3 +56,26 @@ def __str__(self):
 
     def get_absolute_url(self):
         return reverse('event-details', args=[self.starts.year, self.starts.month, self.starts.day, self.slug])
+
+
+class Attendance(models.Model):
```

Comment:
I have thought about this, but currently there is nowhere to use it. If you need a list of attendees, then you need them together with their responses, so just attendees field on event is redundant.